### DISCUSSION TITLE
Parameter to "Show app" on startup?

### DISCUSSION THREAD

**Discussion by Bob-F-xxx** [PurpleI2P/i2pd#1914] [Q&A]
On Windows, is there a way to automatically open the "stats" app when starting i2pd?

**Comment by r4sas** [comment]
There is no such option. Why do you need it?

**Comment by Bob-F-xxx** [comment]
It's not that I really need it.  It would just be convenient if the possibility existed.  i2pd runs on a small headless system, accessed with Remote Desktop and when taking a quick peek if everything runs fine a stats screen would be nice.  Now I need to open it manually after any restart.

**Reply by Vort** [reply]
If small browser can fit into small headless system, you can make i2pd webconsole as home page and autostart browser instead.

**Reply by Bob-F-xxx** [reply]
I know but I'm not going to install a browser just for that :-)  Another way would be to use a macro-tool to automate mouse/key actions.

**Reply by Vort** [reply]
AHK (AutoHotkey) is easy to learn (if you want to go that way).

**Reply by Bob-F-xxx** [reply]
I used MacroExpress in the past but it's a bit overkill for such a simple task.  Will look into it after I've solved a restart issue.  For some reason that system refuses to restart (shutdown /r /f) after a few days.  It works at first but after a week or so it goes to sleep or something in stead doing a reboot.  Very annoying.

**Reply by Vort** [reply]
Never encountered such problem. Maybe system logs have clues?

**Reply by Bob-F-xxx** [reply]
Not a single clue that makes sense,
